rotkehlchen/chain/evm/decoding/cowswap/decoder.py

```diff
@@ -375,6 +375,7 @@ def _decode_normal_claim(self, context: DecoderContext) -> DecodingOutput:
                 amount=amount,
                 asset=self.vcow_token,
                 counterparty=CPT_COWSWAP,
+                airdrop_identifier='cow_mainnet' if self.evm_inquirer.chain_id == ChainID.ETHEREUM else 'cow_gnosis',  # noqa: E501
```

Comment:
1. Move this out of the for.
2. Just doing (1) is probably fine. But perhaps can also instead move it to the constructor as a property of the decoder to not do the if check every time.Though it's a memory vs cpu tradeoff and perhps not worth it?

Reply:
2. it's probably okay to have the if check, because this decoder function will only execute once (only if the user ever claimed it). But, if we pass it through the constructor then it will always occupy memory